.gitignore

```diff
@@ -31,8 +31,8 @@ include/generated/*
 iroha.conf
 peers.list
 cmake-build*
+test/system/irohad_test_data/config.sample.copy
```

Comment:
This file is in the repo and hence is versioned with other source. I think for your purposes you should use `.git/info/exclude`, which is your private and does not synchronize.

Reply:
Ssssssssssss! I had to check this.
But anyway, it looks like a bad test design.